### ISSUE TITLE
One Google account...but two YouTube profiles. Any way to have this app scan both for subscriptions?

### ISSUE THREAD

**Issue by paulescobar** [SPDUK/youtube-subscription-dl#14]
I have one Google account. But that account has two different YouTube profiles. One profile contains my gaming subscriptions. The other contains more news related subscriptions.

When I first ran "yt-subs.sh", the "authorization" steps had me select one of the YouTube profiles. Is there any way I can get your app to include both YouTube profiles?

**Comment by SPDUK** [comment]
@paulescobar  I don't think that would be possible as it uses OAuth to authorize you with that specific youtube profile, I suppose you could set it up twice with 2 different folders, with each one authorized to a different channel, not a perfect solution, but a solution.

It should work, but it would skip repeat videos if both are subscribed to the same channel, which could be seen as a feature not a bug :smiley:
